docs/source/developers/continuous_integration/crossbow.rst

```diff
@@ -117,14 +115,16 @@ to step 3:
    username and ``ghrepo`` is the GitHub repository name (typically
    ``crossbow``):
 
-   -  TravisCI: ``https://travis-ci.org/<ghuser>/<ghrepo>/settings``
-   -  Appveyor:
-      ``https://ci.appveyor.com/project/<ghuser>/<ghrepo>/settings/environment``
-   -  CircleCI:
-      ``https://circleci.com/gh/<ghuser>/<ghrepo>/edit#env-vars``
+   - **CircleCI**: ``https://circleci.com/gh/<ghuser>/<ghrepo>/edit#env-vars``
```

Comment:
I'm not sure we use CircleCI anymore? @assignUser

Reply:
I am not aware of any jobs that use it either (checked tasks.yml too).

Reply:
Thank you for pointing this out, @pitrou and confirming @assignUser . I'll remove CircleCI from this documentation page.